pkg/controller/agent/pod.go

```diff
@@ -388,21 +388,24 @@ func getAssociatedFleetServer(params Params) (commonv1.Associated, error) {
 }
 
 func trustCAScript(ver version.Version, caPath string) string {
-	sharedCAPath := "/etc/pki/ca-trust/source/anchors/"
-	updateCmd := "update-ca-trust"
-	if ver.GTE(version.MinFor(7, 17, 0)) {
-		sharedCAPath = "/usr/local/share/ca-certificates"
-		updateCmd = "update-ca-certificates"
-	}
+	redhatSharedCAPath := "/etc/pki/ca-trust/source/anchors/"
```

Comment:
`ver` argument is unused and should be removed.

Reply:
This was caught in the ci tests, and I removed it earlier.